Automattic/wp-calypso
# Pricing page rework: Lay the foundation for product lists and cards

#### Proposed Changes

* This PR wires up the product list data with the components so that we can proceed to work with Product lists and cards.
* It does not deal with styling for now.
* It also removes unit tests for now because components change frequently, resulting in test failures.

#### Testing Instructions



* Boot up this PR 
    * Run `git fetch && git checkout add/jp-pricing-rework/base-for-product-cards`
    * Run `yarn start-jetpack-cloud`
    * Goto [http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1](http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1)
* Or click on Jetpack Cloud live link below
    * Goto `/pricing?flags=jetpack/pricing-page-rework-v1`
* Confirm that you see "Backup" and "VideoPress" under "Most popular products"
* Confirm that you see the other products under "All products"
* Goto Bundles view
* Confirm that you see "Security" and "Complete" under "Most popular bundles"


#### Pre-merge Checklist






Completes 1202796695664022-as-1202796695664108/f

<img width="1243" alt="Screenshot 2022-08-26 at 3 40 02 PM" src="https://user-images.githubusercontent.com/18226415/186881947-dbc8ed34-673d-450d-9faf-ec8d0dd931c0.png">
<img width="1258" alt="Screenshot 2022-08-26 at 3 40 48 PM" src="https://user-images.githubusercontent.com/18226415/186881936-303814b2-2648-4aab-8644-bc1f664649b9.png">


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202796695664108
